flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/combined/NtTable.java

```diff
@@ -0,0 +1,47 @@
+package com.flash3388.frc.nt.combined;
+
+import edu.wpi.first.networktables.*;
```

Comment:
Don't use global imports, they're problematic

Reply:
it was the IDE I SWEAR

Reply:
You can remove it in _settings_